src/ui/component/filePicker/filePicker.tsx

```diff
@@ -62,32 +54,36 @@ export const FilePicker: FC<FilePickerProps> = ({
         if (!fsEntry) throw new Error('Could not read item')
         if (!file || fsEntry.isDirectory) throw new Error('Dropped item not a file')
 
-        return toFileDTO(file)
+        return toFileDto(file)
       } catch (error: unknown) {
         isError(error) && onError?.(error)
       }
     },
-    [onError, toFileDTO]
+    [onError, toFileDto]
+  )
+
+  const isFileDtoArray = useCallback(
```

Comment:
What's the use case in which a file could be `undefined` into the array?

Reply:
We can drop anything draggable item in the drop zone so we might end up having a variety of non-file elements.